devices/motorola-potter/default.nix

```
@@ -40,18 +63,20 @@
 
   mobile.usb = {
     mode = "gadgetfs";  # Google
-    idVendor = "18D1"; # "POCO F1", distinguish nixos from fastboot/lk2nd
-    idProduct = "4EE7";
+    idVendor = "18D1";
+    idProduct = "4EE7"; # anything but "D001" to distinguish nixos from fastboot/lk2nd
 
     gadgetfs.functions = {
-      rndis = "rndis_bam.rndis";
+      rndis = "rndis_bam.rndis"; # don't know if this is correct
```

This is probably wrong, IIRC `rndis_bam` should be for qualcomm vendor kernels.

I believe you would want, and should use all the default mainline functions, si:

```suggestion
      rndis = "rndis.usb0";
```